tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradePlanner.java

```diff
@@ -37,13 +39,17 @@
 
 	public UpgradePlan loadUpgradePlan(String name);
 
+	public List<UpgradePlan> loadUpgradePlans();
```

Comment:
I don't think the name of loadUpgradePlans is good choose, as it may be confused with loadUpgradePlan(String name)